server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -81,6 +87,7 @@ export function create(
             singleUseTokenCache,
         }),
         throttle(throttler, winston, commonThrottleOptions),
+        throttle(throttler, winston, commonThrottleOptionsCreateDoc),
```

Comment:
I think it would be better to throttle based on cluster limit before tenant limit?

Reply:
How do we decide on the order? Is that because cluster limit throttle might be more possible to happen than tenant limit throttle? Thanks. :)

Reply:
That's correct. If the cluster is reaching its limit, tenantLimit does not matter. So throttling at cluster limit first is better.

Reply:
Got it. Done. :)